### DISCUSSION TITLE
Use Keda as a dependency in my application Helm chart

### DISCUSSION THREAD

**Discussion by simonlsk** [kedacore/keda#2725] [Q&A / Need Help] [answered]
Hello everyone! 
Is it possible to use Keda's Helm chart as a dependency of my application Helm chart, and refer to a CRD in my chart?
Keda's CRDs are not located in the `crds` directory, therefore I cannot add a `TriggerAuthentication` as part of my templates without failing. 

Is there a best practice other than adding Keda's CRD's in my chart or manually installing Keda prior to my application?

I couldn't find anything helpful in the documentation.

Thank you!

**Comment by tomkerkhove** [comment, marked as answer]
We are tracking this in https://github.com/kedacore/charts/issues/226 with an issue open on Hem upstream https://github.com/helm/helm/issues/10585.

The current guidance is to install KEDA before installing your app.
